File: TokanPages.WebApi.Dto/Content/Common/ContentUnsubscribe.cs
using System.Diagnostics.CodeAnalysis;
using Newtonsoft.Json;

namespace TokanPages.WebApi.Dto.Content.Common;

/// <summary>
/// ContentUnsubscribe
/// </summary>
[ExcludeFromCodeCoverage]
public class ContentUnsubscribe
{
    /// <summary>
    /// Caption
    /// </summary>
    [JsonProperty("caption")]
    public string Caption { get; set; } = "";

    /// <summary>
    /// Text1
    /// </summary>
    [JsonProperty("text1")]
    public string Text1 { get; set; } = "";

    /// <summary>
    /// Text2
    /// </summary>
    [JsonProperty("text2")]
    public string Text2 { get; set; } = "";

    /// <summary>
    /// Text3
    /// </summary>
    [JsonProperty("text3")]
    public string Text3 { get; set; } = "";

    /// <summary>
    /// Button
    /// </summary>
    [JsonProperty("button")]
    public string Button { get; set; } = "";
}

File: TokanPages.ClientApp/src/Store/Defaults/Content/contentUnsubscribe.ts
import { ContentUnsubscribeState } from "../../States";

export const ContentUnsubscribe: ContentUnsubscribeState = 
{
    isLoading: false,
    content: 
    {
        language: "",
        contentPre:
        {
            caption: "",
            text1: "",
            text2: "",
            text3: "",
            button: ""
        },
        contentPost:
        {
            caption: "",
            text1: "",
            text2: "",
            text3: "",
            button: ""
        }
    }    
}